src/sql/base/browser/ui/panel/panel.component.ts

```diff
@@ -353,7 +361,9 @@ export class PanelComponent extends Disposable implements IThemable {
 
 	private focusOnTabHeader(index: number): void {
 		if (index >= 0 && index <= this._tabHeaders.length - 1) {
-			this._tabHeaders.toArray()[index].focusOnTabHeader();
+			const tabHeaderArray = this._tabHeaders.toArray();
```

Comment:
I'd suggest adding a comment explaining why this logic is necessary (with a link to the docs you linked earlier too)